optimade/server/config.py

```diff
@@ -94,15 +94,21 @@ class ServerConfig(Config):
 
     @staticmethod
     def _DEFAULTS(field: str) -> Any:
+        from optimade import __version__
+
         res = {
             "use_real_mongo": False,
             "mongo_database": "optimade",
             "links_collection": "links",
             "references_collection": "references",
             "structures_collection": "structures",
             "page_limit": 500,
-            "version": "v0.10.0",
+            "api_version": "0.10.0",
             "default_db": "test_server",
+            "implementation.name": "Example implementation",
```

Comment:
One question here: why use a dot only here and not elsewhere?

I would suggest to either use the underscore or a nested dict

Reply:
Yeah. I am not really proud of this PR...

To make a long story short: The `config.ini` file already has a section called `[IMPLEMENTATION]`, which means I can't do the same as I did for the `[PROVIDER]` section. So I've had to be a bit more ... creative.

It could be done by simply re-shuffling all the existing parameters from under '[IMPLEMENTATION]` to a new and differently named section, but i couldn't come up with a name that fit ...